docs/reference/tctl/workflow/show.md

```diff
@@ -0,0 +1,101 @@
+---
+id: show
+title: tctl workflow show
+description: How to show Workflow History using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../../../content/what-is-a-workflow-execution.md'
+import * as WhatIsAWorkflowId from '../../../content/what-is-a-workflow-id.md'
+import * as WhatIsARunId from '../../../content/what-is-a-run-id.md'
+
+The `tctl workflow show` command shows History for the specified <preview page={WhatIsAWorkflowExecution}>Workflow Execution</preview>.
+
+**Syntax** `tctl workflow show <command options> <arguments...>`
+
+### --workflow_id
+
+Specify the <preview page={WhatIsAWorkflowId}>Workflow Id</preview>.
+
+**Syntax** `--workflow_id <value>`
```

Comment:
Something about the bold + the code style makes this pop more than we want I think.

Maybe try not using bold, or keep the bold and move cmd to code block, or make these bullet points?

Reply:
Yeah, the bold was catching my eye too. If we ditch the "Syntax" run-in heading at the top, we might as well delete it here too. I'll try that and also de-bold the "Alais[es]" headings too.

Reply:
I like the revisions; see what you think. I'm still debating whether to leave "Example" bold. At the moment, I'm going to do so because other examples might begin with explanatory text.